app/services/people/retrieve_court_cases.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module People
+  class RetrieveCourtCases
+    def self.call(person)
+      nomis_court_cases_response = NomisClient::CourtCases.get(person.latest_nomis_booking_id)
+
+      JSON.parse(nomis_court_cases_response).map do |court_case|
```

Comment:
Do we want to order by start date or is the Nomis api always sorting correctly?

Reply:
Talked to @alexdesi about this and we agreed that this isn't necessarily the responsibility of this api/nor necessary for this iteration.